jax/api.py

```diff
@@ -107,8 +107,8 @@ def jit(fun, static_argnums=(), device=None, backend=None):
       Optional, the Device the jitted function will run on. (Available devices
       can be retrieved via ``jax.devices()``.) The default is inherited from
       XLA's DeviceAssignment logic and is usually to use ``jax.devices()[0]``.
-    backend: This is an experimental feature and the API is likely to change.
```

Comment:
I think that we should get in the habit of deprecating arguments from public APIs before removing them. I would suggest keeping the `backend`, immediately turning it into device=devices(backend)[0], issuing in awarning, adding the note to Changelog, adding a reminder somewhere to remove it in a future release. This is a lot more expensive than just removing it, but I think that as we have more and more users it is a good thing to do. 

Also, it may make sense to do codesearch/ for people who use the backend argument